src/data-loaders/RelationalDataLoader.js

```diff
@@ -94,12 +94,40 @@ const convertPostgresSQLResponseToColumnMetaData = (rows) => {
       label: row.name,
       name: row.name,
       type: row.dataTypeID,
-      typeName,
+      typeName: typeName ? typeName.toUpperCase() : 'UNKOWN',
     };
   });
 };
+
+const injectVariables = (statement, variableMap) => {
+  const result = Object.keys(variableMap).reduce((statmnt, key) => {
+    if (
+      variableMap[key] === null ||
+      variableMap[key] === false ||
+      variableMap[key] === true
+    ) {
+      return statmnt.replace(key, `${variableMap[key]}`);
+    }
+    // @TODO: Differentiate number from string inputs...
+    return statmnt.replace(key, `'${variableMap[key]}'`);
+  }, statement);
+  return result;
+};
+
 const executeSqlStatements = async (client, req) =>
-  Promise.mapSeries(req.statements, (statement) => client.query(statement));
+  Promise.mapSeries(req.statements, async (statement) => {
+    if (req.variableMap) {
```

Comment:
would it be possible to please move this `if` condition inside `injectVariables`